### PR TITLE
Test replicator

### PR DESCRIPTION
**Pull request by kosssi**

### CODE REVIEW

File: fastlane/metadata/android/en-US/full_description.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Cozy.io is a platform that brings all your web services in the same private space. With it, your web apps and your devices can share data easily, providing you with a new experience. You can install Cozy on your own hardware where no one profiles you.
+
+This is the Android mobile client to your personnal Cozy server. Connect it to your Cozy server and browse through your cozy files from your Android device. You can also synchronize your contacts and your calendars.
+
+Use github issues to report bugs and vote for next features : https://github.com/mycozycloud/cozy-files-mobile/issues
```
Comment (nono) [inline fastlane/metadata/android/en-US/full_description.txt]: @kosssi the URL looks wrong

File: src/app/replicator/main.coffee [hunk 2]
```diff
@@ -164,6 +51,24 @@ module.exports = class Replicator extends Backbone.Model
...
+                body:
+                    include_docs: true
+                    show_revs: true
+                retry: doc.retry
+            @requestCozy.request options, (err, res, rows) ->
+                if not err and res.statusCode isnt 200
+                    err = new Error res.statusCode, res.reason
+
+                return callback err if err
+                callback null, rows
```
Comment (nono) [inline src/app/replicator/main.coffee]: You can just `callback err, rows` instead of having these 2 lines

File: src/app/replicator/main.coffee [hunk 3]
```diff
@@ -368,3 +282,97 @@ module.exports = class Replicator extends Backbone.Model
         DeviceStatus.checkReadyForSync (err, ready, msg) =>
             return callback() unless ready
             @fileCacheHandler.downloadUnsynchronizedFiles callback
+
+
+# BEGIN Login Function: TODO move in lib/login.coffee
```
Comment (nono) [inline src/app/replicator/main.coffee]: You don't use cozy-device-sdk for that?

**Comment by nono** [comment]
@kosssi what does it mean that the codecov/patch check failed? Is it because you added a lot more code than tests?

File: src/app/replicator/main.coffee (hunk 3 above)
Comment (kosssi) [inline src/app/replicator/main.coffee]: not yet...

**Comment by nono** [comment]
@kosssi don't forget that PR

**Comment by nono** [comment]
@kosssi what do we do with this PR?

File: fastlane/metadata/android/en-US/full_description.txt (hunk 1 above)
Comment (kosssi) [inline fastlane/metadata/android/en-US/full_description.txt]: 👍

File: src/app/replicator/main.coffee (hunk 2 above)
Comment (kosssi) [inline src/app/replicator/main.coffee]: 👍

**Pull request merged**